bedevere/bpo.py

```diff
@@ -1,5 +1,6 @@
 """Check if a bugs.python.org issue number is specified in the pull request's title."""
 import re
+import aiohttp
```

Comment:
```suggestion

import aiohttp
```

Reply:
Should there be an empty line between `import aiohttp` and `from gidgethub import routing`?

Reply:
This hasn't been fixed yet.

Reply:
There is now a blank line between `import re` and `import aiohttp`. Should there be a blank line between `import aiohttp` and `from gidgethub import routing`?

Reply:
Please follow https://www.python.org/dev/peps/pep-0008/#imports.

Reply:
I'm terribly sorry, but I can't see how the updated version does not follow PEP8. On the GitHub 'Files changed' interface there is indeed a blank line between `stdlib` and third parties `import`s, which is what PEP8 specifies. It is also consistent with how `__main__.py` does `imports, so I'm not exactly sure what I should change.

Reply:
So sorry, you're right. I've received a flood or PRs this week and I was thinking about the gidgethub repo. 😄